analyses/cell-type-dsrct/README.md

```diff
@@ -0,0 +1,43 @@
+# DSRCT Cell Type analysis module
+
+
+
+## Description
+
+This analysis aims to annotate the DSRCT samples in the SCPCP000013 (n=7) dataset.
+
+To do so, we will:
+
+- Assess the quality of the data set
+- Curate a list of marker genes associated with DSRCT cells
+- Detect the expression levels of DSRCT marker genes in the data set
+- Identify clusters in the DSRCT samples
+- Use the list of marker genes to identify tumor cells
+- Perform copy number inference to identify tumor cells
+- Annotate normal cells
+- Perform clustering on tumor cells and identify tumor cell states
+
+## Usage
+
+The code for the module will be in the form of a notebook.
+
+## Input files
+
+
+The input is dependent on the output fles run from `download-data.py`. This generates `SingleCellExperiment`  for DSRCT samples.
```

Comment:
Can you include the command for downloading the data files specific to this analysis? I assume for this project that it would look like this? 
````suggestion
The input files are the `SingleCellExperiment` files for project SCPCP000013, which can be downloaded with the following command:

```
 ./download-data.py --project SCPCP000013
```

````

As you do not need all of these files for this project, you could specify the specific 7 samples instead, but that is really up to you, as the total project is not too large.